operator/pkg/ciliumendpointslice/manager.go

```diff
@@ -76,12 +76,17 @@ func (c *cesManager) createCES(name, ns string) CESName {
 // CES object or updating an existing CES object.
 func (c *cesManager) UpdateCEPMapping(cep *cilium_v2.CoreCiliumEndpoint, ns string) []CESKey {
 	cepName := GetCEPNameFromCCEP(cep, ns)
-	c.logger.Debug("Insert CEP in local cache", logfields.CEPName, cepName.string())
+	c.logger.Debug("Insert CEP in local cache",
+		logfields.CEPName, cepName.string(),
```

Comment:
A bit tangential to this change, but here we should probably make `string` a proper stringer implementation, and let slog deal with the conversion, to save the operations when debug logs are disabled.